homeassistant/components/websocket_api/http.py

```diff
@@ -222,7 +222,7 @@ def handle_hass_stop(event):
                 self._to_write.put_nowait(None)
                 # Make sure all error messages are written before closing
                 await self._writer_task
-            except asyncio.QueueFull:
+            except:
```

Comment:
Only catch the exceptions that can be thrown instead of catching all exceptions. This will also solve the flake8 and pylint errors.
```suggestion
            except (asyncio.QueueFull, asyncio.exceptions.CancelledError):
```